replica/table.cc

```diff
@@ -2532,6 +2532,11 @@ table::table(schema_ptr schema, config config, lw_shared_ptr<const storage_optio
         tlogger.warn("Writes disabled, column family no durable.");
     }
 
+    // we initialize 0 for 30 seconds, because those values are used in DescribeTable
```

Comment:
When code in table.cc needs to mention Alternator API, it's a sign maybe the code is in the wrong place...